src/rust/ide/file-manager/mock-server/Main.enso

```diff
@@ -0,0 +1,19 @@
+vmain =
+    x = 6
+    y = x.foo 5
+    z = y + 5
+    z
+        
+Number.foo = x ->
+    y = this + 3
+    z = y * x
+    z
+        
+encode = x -> x.to_text
+
+incAndEncode = x -> here.encode x+1s
+
+
+#### METADATA ####
+[[{"index":{"value":16},"size":{"value":1}},"3d85f3e2-017a-4558-9e3d-9b2cb096ee1d"],[{"index":{"value":26},"size":{"value":7}},"00779024-2104-4b59-bcb3-1b8cb9af7020"],[{"index":{"value":42},"size":{"value":5}},"741961ef-20b1-4c09-829b-ee1179884364"],[{"index":{"value":52},"size":{"value":1}},"10f003d1-dad4-4bcf-a4cc-69c1a1ed51e7"],[{"index":{"value":7},"size":{"value":55}},"5aecf4f9-3262-40eb-b7a9-21ec466b3381"],[{"index":{"value":89},"size":{"value":8}},"de3d3651-7e78-4d02-a988-1b5cb16f8d00"],[{"index":{"value":106},"size":{"value":5}},"f0bc24d2-35de-4ad9-86d3-a0c9a4edbd94"],[{"index":{"value":116},"size":{"value":1}},"bc092512-36d1-4461-9d21-6a920ad67e6d"],[{"index":{"value":76},"size":{"value":50}},"6fecfcf0-00eb-4288-9296-267da6814c33"],[{"index":{"value":136},"size":{"value":14}},"04ea0750-2283-45c1-8804-1da4a4f1a3f0"],[{"index":{"value":167},"size":{"value":21}},"94052370-b277-4e39-bb28-483a502b0c4d"]]
+{"ide":{"node":{"d951b3e6-35f6-4e52-bae7-53711a79331c":{"position":{"vector":[64.0,-25.0]}},"7e8e8326-9231-436b-9c6b-b1a30cb2dd22":{"position":{"vector":[363.0,23.0]}}}}}
```

Comment:
This file should not be committed. Should it?

Reply:
It shouldn't! I removed it. :)